<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="pedir2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Codigo </string>
    </property>
   </widget>
   <widget class="QComboBox" name="pedir1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  rgb(0,126,249);
border-radius: 8px;
color:rgb(255, 255, 255);
background-color:rgb(24, 30, 50);
font: 75 12pt&quot;Times New Roman&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Minorista</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mayorista</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_nombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Playera</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cotizar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cotizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
